dev-libs/crypto++/additional-files/libcrypto++.pc

```diff
@@ -0,0 +1,10 @@
+# Written by Alexander Rødseth <rodseth@gmail.com>
```

Comment:
You can do it in your recipe too, it doesn't need an extra file.

Reply:
Example: https://github.com/haikuports/haikuports/blob/409b5225c23081d9ca44e35f45871791bee5c4cd/app-arch/snappy/snappy-1.1.7.recipe